File: android/app/src/main/java/com/yourapp/ReactActivityDelegate.java
package com.yourapp;

import com.facebook.react.ReactActivityDelegate;
import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;
import android.app.Activity;

public class ReactActivityDelegate extends ReactActivityDelegate {
    public ReactActivityDelegate(Activity activity, String mainComponentName) {
        super(activity, mainComponentName);
    }

    @Override
    protected RNGestureHandlerEnabledRootView createRootView() {
        return new RNGestureHandlerEnabledRootView(getActivity());
    }
}

File: android/app/src/main/kotlin/com/myfabricapp/ReactActivityDelegate.kt
package com.myfabricapp

import com.facebook.react.ReactActivityDelegate
import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView

class MyReactActivityDelegate(activity: MainActivity) : ReactActivityDelegate(activity, activity.getMainComponentName()) {
    override fun createRootView(): RNGestureHandlerEnabledRootView {
        return RNGestureHandlerEnabledRootView(activity)
    }

    override fun getMainComponentName(): String {
        return activity.getString(R.string.app_name)
    }
}